src/main/java/com/Model/Event.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Author: Zhe
+ * Date: May30-2020
+ * Event class is going to store the details of patient attended events
+ */
+package com.Model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Date;
```

Comment:
java.util.Date seems mostly deprecated. I don't mind keeping it, but be ready for an alternative should we need one!

Reply:
I change the type of date from Date to long.

Reply:
long loses the information that this is a Date. There are more recent Java classes that have replaced Date. See Calendar or LocalDate for example.

Reply:
I think it's better to keep java.util.Date than replace it with long in any case.

Reply:
I will keep date and Use Calendar to create specific date and use 
calendar.getDate to get Date object.